apps/E2E/src/Checkbox/specs/Checkbox.spec.win.ts

```diff
@@ -26,18 +25,35 @@ describe('Checkbox Accessibility Testing', () => {
     await CheckboxPageObject.scrollToTestElement();
   });
 
-  it('Checkbox - Validate accessibilityRole is correct', async () => {
-    await expect(await CheckboxPageObject.getAccessibilityRole()).toEqual(CHECKBOX_A11Y_ROLE);
+  it('Validate "accessibilityRole" defaults to Checkbox "ControlType" element attribute.', async () => {
+    await expect(
+      await CheckboxPageObject.compareAttribute(CheckboxPageObject._primaryComponent, Attribute.AccessibilityRole, CHECKBOX_A11Y_ROLE),
+    ).toBeTrue();
+
     await expect(await CheckboxPageObject.didAssertPopup()).toBeFalsy(CheckboxPageObject.ERRORMESSAGE_ASSERT);
   });
 
-  it('Checkbox - Set accessibilityLabel', async () => {
-    await expect(await CheckboxPageObject.getAccessibilityLabel(ComponentSelector.Primary)).toEqual(CHECKBOX_ACCESSIBILITY_LABEL);
+  it('Set "accessibilityLabel" prop. Validate "accessibilityLabel" value propagates to "Name" element attribute.', async () => {
+    await expect(
+      await CheckboxPageObject.compareAttribute(
+        CheckboxPageObject._primaryComponent,
+        Attribute.AccessibilityLabel,
+        CHECKBOX_ACCESSIBILITY_LABEL,
+      ),
+    ).toBeTrue();
+
     await expect(await CheckboxPageObject.didAssertPopup()).toBeFalsy(CheckboxPageObject.ERRORMESSAGE_ASSERT);
   });
 
-  it('Checkbox - Do not set accessibilityLabel -> Default to Checkbox label', async () => {
-    await expect(await CheckboxPageObject.getAccessibilityLabel(ComponentSelector.Secondary)).toEqual(CHECKBOX_TEST_COMPONENT_LABEL);
+  it('Do NOT set "accessibilityLabel" prop. Validate "Name" element attribute defaults to the checkbox label.', async () => {
+    await expect(
+      await CheckboxPageObject.compareAttribute(
+        CheckboxPageObject._secondaryComponent,
+        Attribute.AccessibilityLabel,
+        CHECKBOX_TEST_COMPONENT_LABEL,
+      ),
+    ).toBeTrue();
```

Comment:
Why use `toBeTrue()` here and `toBeTruthy()` below? What's the difference?

Reply:
I explain it here: https://github.com/microsoft/fluentui-react-native/pull/2480#discussion_r1061023605

In this case, it might be me getting trigger happy on VSCode autocomplete.

Reply:
Update: `.toBeTruthy` allows you to pass an error message on fail whereas `.toBeTrue` does not. For methods using waiters, it doesn't matter since the error should be passed into the waiter itself, but for everything else, `.toBeTruthy` provides a lot of utility.

Reply:
Let's be consistent across the repo and use `.toBeTruthy()` everywhere. It provides the same functionality as `.toBeTrue()` but with the extra error message feature.